src/components/Reports/LostTime/AddLostTime.jsx

```diff
@@ -285,15 +288,15 @@ const AddLostTime = props => {
   };
 
   return (
-    <Modal isOpen={props.isOpen} toggle={props.toggle}>
-      <ModalHeader toggle={props.toggle}>
+    <Modal isOpen={props.isOpen} toggle={props.toggle} className={darkMode ? 'text-light dark-mode' : ''}>
+      <ModalHeader toggle={props.toggle} className={darkMode ? 'bg-space-cadet text-light' : ''}>
```

Comment:
Since we already have the classname, 'text-light', within the parent component, it is unnecessary for us to declare it again on ModalHeader, ModalBody, or ModalFooter